lib/src/git.rs

```diff
@@ -685,6 +685,12 @@ pub fn reset_head(
 ) -> Result<(), git2::Error> {
     let first_parent_id = &wc_commit.parent_ids()[0];
     if first_parent_id != mut_repo.store().root_commit_id() {
+        if let Some(git_head) = mut_repo.view().git_head().as_normal() {
+            if git_head == first_parent_id {
+                return Ok(());
```

Comment:
indeed, thanks for the suggestion :)

IIUC we're now using just `mut_repo.view().git_head()` which returns a reference, so that leads me to a small question about the preferred style: is `*ref == val` any better than `ref == &val` for the comparison?

Reply:
I don't have a preference. Maybe Yuya or someone else does

Reply:
I have no preference. The choice is always arbitrary.